clippy_lints/src/misc.rs

```diff
@@ -48,8 +48,8 @@ declare_clippy_lint! {
 declare_clippy_lint! {
     /// **What it does:** Checks for comparisons to NaN.
     ///
-    /// **Why is this bad?** NaN does not compare meaningfully to anything – not
-    /// even itself – so those comparisons are simply wrong.
+    /// **Why is this bad?** NaN does not compare meaningfully to anything   not
+    /// even itself   so those comparisons are simply wrong.
```

Comment:
It looks like the `-` was replaced with a space here, was this on purpose?

Reply:
No, that wasn't intentional.  When I save that file with my editor configured to run rustfmt, it replaces the en dashes with spaces.  :(  I've pushed a commit to restore them.

Reply:
I was a victim of rust-lang/rust-mode#279.